src/scattering.F90

```diff
@@ -419,6 +450,19 @@ subroutine nu_scatter_elastic_heavy_total(neutrino_energy,transport,lepton,eos_v
   
   !integrate over phi
   crosssection = 2.0d0*pi*crosssection
+  
```

Comment:
For coherent scattering (CE\nuNS) \delta = 1 and the non-transport cross section is 3/2 of the transport cross section,  So this can be worked into this fairly easily (i.e. you only have to do the integration once)

Reply:
This ignores the ion-ion correlation issue (that the correction factors were determined for the transport opacity, but I think this is safe to ignore)

Reply:
Done (with comment in code).